canonical-websites/www.ubuntu.com
# Update  plans and pricing page with pricing and services updates

## Done

- Updated the plans and pricing page per the recent beta updates

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/support/plans-and-pricing](http://0.0.0.0:8001/support/plans-and-pricing)
- compare to the [copy doc](https://docs.google.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit#)

## Issue / Card

Fixes #3492
